### PR TITLE
Wikipedia Cleaner

### PR DESCRIPTION
**Pull request by nicoring**
Adds wikipedia cleaner class that uses a precleaned wikipedia corpus and filters out lines with less than `min_length` tokens.

### CODE REVIEW

File: goethe/clean/wikipedia.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class WikipediaCleaner(Cleaner):
+    """Iterates over Wikipedia corpus file and filteres out lines with a length
+       less than `min_length`. We use the corpus from:
+       http://linguatools.org/tools/corpora/wikipedia-monolingual-corpora/
+       The passed corpus file must be cleaned with the Per scripts provided on
+       this website (we used the -nomath and -notables options).
+    """
+
+    def __init__(self, path, min_length=6):
```
Comment (rshkv) [inline goethe/clean/wikipedia.py:12]: Are you sure `6` is enough? Can't think of a sentence other than: _Er ist._

**Comment by rshkv** [comment]
Can you give an estimate of how big the corpus is?

**Comment by rshkv** [comment]
lgtm

File: goethe/clean/wikipedia.py (hunk 1 above)
Comment (nicoring) [reply goethe/clean/wikipedia.py:12]: What do you mean with enough?

**Comment by nicoring** [comment]
The corpus without filtering contains 30.997.699 lines.

**Pull request merged**